bigbluebutton-html5/imports/ui/local-collections/abstract-collection/abstract-collection.js

```diff
@@ -0,0 +1,164 @@
+import SubscriptionRegistry from '/imports/ui/services/subscription-registry/subscriptionRegistry.js';
+
+class AbstractCollection {
+  constructor(serverCollection, localCollection, options = {
+    /*
+    the current options:
+    fields: the field to be used in the selects to validate the data, update and remove.
+
+    secondarySubscriptions: set if theres more than one susbcription
+    per collection to avoid conflicts in data set.
+    */
+  }) {
+    this.serverCollection = serverCollection;
+    this.localCollection = localCollection;
+
+    this.lastSubscriptionId = '';
+    this.options = options;
+    this.ignoreDeletes = false;
+    this.createObserver();
+  }
+
+  buildSelector(document) {
+    const { fields } = this.options;
+    const selector = {};
+
+    if (fields) {
+      fields.forEach((field) => {
+        selector[field] = document[field];
+      });
+    } else if (!fields) {
+      return null;
+    }
+
+    return selector;
+  }
+
+  // Replicates remote collection to local collection ( avoiding cleanup during the forced resync )
+  createObserver() {
+    const self = this;
+    // We added an websocket listener because
+    // the obseve makes a poll to get what changed in the collection
+    // So, when the new publication is receive and added to collection
+    // and imediatly removed from collection it's not notified to observer
```

Comment:
Are you saying that the default publication was not reaching the observer (unintentional) OR that the newly added poll response does not trigger  the observer (intentional) OR something else?